depends/packages/qt.mk

```diff
@@ -146,6 +147,7 @@ $(package)_config_opts_linux += -xcb
 ifneq ($(LTO),)
 $(package)_config_opts_linux += -ltcg
 endif
+$(package)_config_opts_freebsd := $$($(package)_config_opts_linux)
```

Comment:
_(comment in random place in the diff, not to clutter the PR main thread and to make it easier to follow replies)_

Would be good to have CI build this to make sure it does not rot.

Reply:
I'm going to add it to https://github.com/hebasto/bitcoin-core-nightly.